### ISSUE TITLE
Using [Component].extend.attrs()` does not trigger highlighting

### ISSUE THREAD

**Issue by ferdaber** [styled-components/vscode-styled-components#63]
This is probably an edge case but it'd be nice to have the following syntax trigger highlighting in the extension:

```jsx
const MyButton = styled.button`
  // this is highlighted correctly
  /* ... */
`

const MyRedButton = MyButton.extend`
  // this is highlighted correctly
  /* ... */
`

const MyBlueButton = MyButton.extend.attrs({ 
  color: 'blue' 
})`
  // this has no highlighting
  /* additional styles... */
`
```

I admit that this is a strange example since the additional styles can be added in the base Button component via the same prop, my actual use case is adding different class names to different extended styled components for backwards compatibility in our component library.

**Comment by aliustaoglu** [comment]
For same example I can see code highlighting is now working but intellisense is not 

I am using: 
* vscode 1.22.2
* vscode-styled-components: 0.0.17
